Dockerfile.test

```diff
@@ -0,0 +1,35 @@
+FROM datajoint/djlab:py3.8-debian
+
+USER root
+RUN apt-get update -y
+RUN apt-get install git -y
+
+USER anaconda
```

Comment:
```suggestion
USER anaconda:anaconda
```

Reply:
Git will now be installed through the `apt_requirements.txt` (see [DataJoint image notes](https://datajoint.github.io/miniconda3-docker/#features)).